charts/newrelic-logging/values.yaml

```diff
@@ -23,6 +23,7 @@
 # whether your account is for the EU region or not.
 #
 # endpoint: https://log-api.newrelic.com/log/v1
```

Comment:
Can you change this also to **loggingEndpoint: https://log-api.newrelic.com/log/v1** like you are doing below.